backend/config/config.py

```diff
@@ -0,0 +1,34 @@
+"""
+Configuration settings for the Flask application
+"""
+import os
+
+class Config:
+    """Base configuration"""
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'dev-secret-key'
```

Comment:
Since this is mostly being run locally, are these KEYs currently just placeholders, I assume?  I don't immediately seem them defined anywhere.